source-map-uploader-plugin.js

```diff
@@ -79,14 +79,19 @@ class BugsnagSourceMapUploaderPlugin {
             return null
           }
 
+          let url = '' +
+            // ensure publicPath has a trailing slash
+            publicPath.replace(/[^/]$/, '$&/') +
+            // remove leading / or ./ from source
+            source.replace(/^\.?\//, '')
+
+          // replace parent directory references with empty string
+          url = url.replace(/\.\.\//g, '')
```

Comment:
more on this [here](https://github.com/bugsnag/webpack-bugsnag-plugins/issues/3#issuecomment-2424733506)